developer-information.md

```diff
@@ -16,17 +19,12 @@
     ├── gpc-api-facade              # GPC API Facade
     └── mhs-adaptor-mock            # Dockerfile and required files for mock of MHS Adaptor
 
-## Snomed CT Database
-Please make sure to load the latest release of Snomed CT UK Edition. See [Configuring the SNOMED Database](./OPERATING.md#populating-the-snomed-database) and [snomed-database-loader](/snomed-database-loader/README.md) for more information.
-
 ## Local development
 ### How to start local environment
 1. Go to `docker` directory
 2. Create a copy of `example.vars.sh`, name it `vars.sh`
 3. Fill in the passwords inside `vars.sh` file:
-    - POSTGRES_PASSWORD: Password to be set for the user used to run migrations. It will also be the password for the default postgres user.
-    - GPC_FACADE_USER_DB_PASSWORD: Password for the user connecting to the database in the GPC API Facade module.
-    - GP2GP_TRANSLATOR_USER_DB_PASSWORD: Password for the user connecting to the database in the GP2GP Translator module.
+    - `SNOMED_CT_TERMINOLOGY_FILE`: Path to where your SNOMED ZIP file is downloaded to.
```

Comment:
Is point 3 still valid given that the `POSTGRES_PASSWORD`, `GPC_FACADE_USER_DB_PASSWORD` and `GP2GP_TRANSLATOR_USER_DB_PASSWORD` have been removed?

Maybe `Update vars.sh to include SNOMED terminology file path:`